src/main/java/Task.java

```diff
@@ -0,0 +1,40 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public Task(String description, boolean status) {
+        this.description = description;
+        this.isDone = status;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "✓" : "✘"); //return tick or X symbols
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public String getDescription() {
+        return this.description;
+    }
+
+    @Override
+    public String toString() {
+        return "[" + this.getStatusIcon() + "] " + this.getDescription();
+    }
+
+    public String toText() {
+        return toText("T");
+    }
+
+    public String toText(String type) {
+        int doneInt = this.isDone ? 1 : 0;
```

Comment:
It might be better to use Enum to represent 1 and 0, as it might look like "magic numbers".